### PR TITLE
updating dependences

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
as requested in https://github.com/openmicroscopy/openmicroscopy/pull/4283 this should install appropriate Django version and gunicorn

### CODE REVIEW

File: definitions/ubuntu-14.04-amd64-omerobase/omero_deps.sh [hunk 1]
```diff
@@ -14,9 +14,10 @@ apt-get -y install \
     ice-services python-zeroc-ice \
     postgresql \
     nginx \
+    gunicorn \
 
 # OMERO Python requirements that are newer than distro versions
-pip install django
+pip install "Django>1.8,<1.9"
```
Comment (manics) [inline definitions/ubuntu-14.04-amd64-omerobase/omero_deps.sh]: Just `Django<1.9` should be enough

Comment (aleksandra-tarkowska) [inline definitions/ubuntu-14.04-amd64-omerobase/omero_deps.sh]: but we want explicit 1.8 family, if you have 1.7 or lower requirements will be satisfied.

Comment (manics) [inline definitions/ubuntu-14.04-amd64-omerobase/omero_deps.sh]: This is only used for the vbox image

**Comment by aleksandra-tarkowska** [comment]
Build is green now https://ci.openmicroscopy.org/job/OMERO-VIRTUALBOX-build/

**Comment by sbesson** [comment]
Tested the VBox in https://ci.openmicroscopy.org/job/OMERO-VIRTUALBOX-build/373/consoleText. Works as expected. Web is correctly started.

**Pull request merged**